File: Assignment13/program3.c

// Print all odd numbers up to N

#include <stdio.h>

void Print_Numbers(int Limit)
{
    int i = 0;

    for(i = 1; i <= Limit; i++)
    {
        if(i % 2 != 0)
        {
            printf("%d\n",i);

        }
    }
}

int main()
{
    int Value = 0;

    printf("Enter the Value of N : ");
    scanf("%d",&Value);

    Print_Numbers(Value);

    return 0;
}

File: Assignment18/program3.java

// 3. Write a program to print all odd numbers up to N.

package Assignment18;

import java.util.Scanner;

public class program3 
{
    public void OddNumbers(int iValue)
    {
        int i = 0;

        for(i = 1; i <= iValue; i++)
        {
            if(i % 2 != 0)
            {
                System.out.println(i);
            }
        }
    }

    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        program3 pobj = new program3();

        int iValue = 0;

        System.out.println("Enter the Number : ");
        iValue = sobj.nextInt();

        pobj.OddNumbers(iValue);

        sobj.close();
    }
}
